extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md

```diff
@@ -0,0 +1,129 @@
+4. The paywall has a templated UX that allows for key messages to be customized by the publisher.
+5. Depending on publisher configuration, the dialog can support three different ways to progress:
+    - Sign-in with publisher’s sign-in flow (i.e. publisher provided sign-in page).
+    - Subscribe directly via the publisher (i.e. via a publisher provided subscription page).
+    - If enabled by the publisher, subscribe via [Subscribe with Google](https://developers.google.com/news/subscribe).
+6. If the user completes any of the above authentication flows, the user is returned to the Story page, the paywall is dismissed and the user can consume the rest of the Story.
+7. If the user encounters a subsequent paywalled Story from this publisher, they will continue to retain access based on [amp-subscription mechanisms](https://amp.dev/documentation/components/amp-subscriptions/#combining-the-amp-reader-id-with-publisher-cookies).
+
+# Subscriptions JSON Configuration Example
+
+```html
+...
+<script type="application/ld+json">
+     {
+       "@context": "http://schema.org",
+       "@type": "NewsArticle",
+       "isAccessibleForFree": "False",
+       "isPartOf": {
+         "@type": ["CreativeWork", "Product"],
+         "name" : "Product A",
+         "productID": "publisher.domain.com:product"
+       }
+     }
+</script>
+<script type="application/json" id="amp-subscriptions">
+     {
+       "services": [
+         {
+           "authorizationUrl": "https://publisher.domain.com/entitlement",
+           "pingbackUrl": "https://publisher.domain/pingback",
+           "actions":{
+             "login": "https://publisher.domain.com/signin",
+             "subscribe": "https://publisher.domain.com/subscribe"
+           }
+         },
+         {
+           "serviceId": "subscribe.google.com"
+         }
+       ]
+     }
+</script>
+...
+```
+
+See details about specifying this configuration in [`amp-subscriptions`](https://amp.dev/documentation/components/amp-subscriptions/).
+
+# `amp-story` and `amp-story-subscriptions` attributes
+
+```html
+<amp-story publisher-logo-src='https://publisher-domain.com/logo.jpg'>
+    <amp-story-page>...</amp-story-page>
+    ...
+    <amp-story-subscriptions subscriptions-page-index='3' price='$0.50/week' headline='Culture at your fingertips' description='Subscribe for unlimited access.' additional-description='Cancel anytime.'>
+    </amp-story-subscription>
+</amp-story>
+```
+
+### `publisher-logo-src` {string} required
+
+A logo URL to show the publisher’s subscribe button.
+
+### `subscriptions-page-index` {number} optional
+
+The index of the page that would show the subscriptions dialog. If not specified, it defaults to 2, which is the 3rd page of the story. Note that this specified page cannot be the first two pages nor the last page of the story.
```

Comment:
```suggestion
The index of the page that displays the subscriptions dialog. If not specified, it defaults to 2, which is the 3rd page of the story. Note that this specified page cannot be the first two pages nor the last page of the story.
```

Reply:
Can we explain more what what mean by index? How do they find which index it is or specify it?

Reply:
Done